java-file-formats/word-library/mail-merge/mail-merge-for-group.md

```diff
@@ -23,7 +23,7 @@ The region where the Mail merge operations are to be performed must be marked by
   
 For example, consider that you have a template document as shown.
 
-![Mail merge for a group](../MailMerge_images/Group_mail_merge_template.png)
+![Java File Formats Word Library Mail merge for a group Template](../mailmerge_images/java-file-formats-word-library-mail-merge-template.png)
```

Comment:
Don't use file-format in image name. Change the image name as per the below PR comments.
https://github.com/syncfusion-content/java-file-format-docs/pull/113#pullrequestreview-1474402940

Reply:
Image name changed